backend/development.ini.sample

```diff
@@ -1,267 +1,373 @@
-###
-# app configuration
-# https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html
-###
-[pipeline:main]
-pipeline = tracim_web
-[app:tracim_web]
-use = egg:tracim_backend
-
-# hapic debug mode: return python traceback of error
-debug = True
-
-pyramid.reload_templates = true
-pyramid.debug_authorization = false
-pyramid.debug_notfound = false
-pyramid.debug_routematch = false
-pyramid.default_locale_name = en
-#pyramid.includes =
-#    pyramid_debugtoolbar
+######################
+# Tracim Config File #
+######################
 
-[pipeline:webdav]
-pipeline = tracim_webdav
-[app:tracim_webdav]
-use = egg:tracim_backend#webdav
+## GENERAL TRACIM config:
+# These config are common for all app : REST API, Webdav, etc...
 
 [DEFAULT]
-sqlalchemy.url = sqlite:///%(here)s/tracim.sqlite
 
-retry.attempts = 3
-
-# By default, the toolbar only appears for clients from IP addresses
-# '127.0.0.1' and '::1'.
-# debugtoolbar.hosts = localhost ::1
+###############
+# BASIC SETUP #
+###############
+
+# easy setup (shortcut) for setting most common config
+# search for "%(param)s" to check where those param are used and why
+
+## global info (need for email and links)
+basic_setup.website_title = TRACIM
+basic_setup.website_base_url =  http://localhost:6543
+basic_setup.listen = localhost:6543
+## storage path/connection
+basic_setup.sqlalchemy_url = sqlite:///%(here)s/tracim.sqlite
+basic_setup.depot_storage_dir = %(here)s/depot/
+basic_setup.preview_cache_dir = /tmp/tracim/preview/
```

Comment:
I'm not sure it is a good default storage dir:

* Generated preview files have not to be deleted, once generated they will be able to be used. Maybe it is a problem about space disk storage ? It easier for user to avoid space disk saturation ? I would like have your opinion @lebouquetin and @PhilippeAccorsi .

Reply:
After discuss with @lebouquetin we leave this parameter with this default path